apps/src/p5lab/spritelab/libraries/PoemBotLibrary.js

```diff
@@ -99,47 +99,154 @@ export default class PoemBotLibrary extends CoreLibrary {
       },
 
       setPoem(key) {
-        if (key === 'wordsworth') {
-          this.poemState = {
-            ...this.poemState,
-            title: 'I Wandered Lonely as a Cloud',
-            author: 'William Wordsworth',
-            lines: [
-              'I wandered lonely as a cloud',
-              "That floats on high o'er vales and hills,",
-              'When all at once I saw a crowd,',
-              'A host, of golden daffodils;',
-              'Beside the lake, beneath the trees,',
-              'Fluttering and dancing in the breeze.'
-            ]
-          };
-        } else if (key === 'dickinson') {
-          this.poemState = {
-            ...this.poemState,
-            title: 'If I can Stop one Heart from Breaking',
-            author: 'Emily Dickinson',
-            lines: [
-              'If I can stop one heart from breaking,',
-              'I shall not live in vain;',
-              'If I can ease one life the aching,',
-              'Or cool one pain,',
-              'Or help one fainting robin',
-              'Unto his nest again,',
-              'I shall not live in vain.'
-            ]
-          };
-        } else if (key === 'silverstein') {
-          this.poemState = {
-            ...this.poemState,
-            title: 'Batty',
-            author: 'Shel Silverstein',
-            lines: [
-              'The baby bat',
-              'Screamed out in fright',
-              "'Turn on the dark;",
-              "I'm afraid of the light.'"
-            ]
-          };
+        switch (key) {
```

Comment:
nit: pull the poems (author, title, lines) into an object constant to make this more readable. then `setPoem` could become:

```javascript
setPoem(key) {
  const poem = POEMS[key];
  if (poem) {
    this.poemState = {...this.poemState, ...poem};
  }
}
```